arch/arm64/boot/dts/qcom/sdm632-motorola-ocean.dts

```diff
@@ -21,6 +21,9 @@
 
 	backlight: backlight {
 		compatible = "led-backlight";
+		brightness-levels = <0 16 32 48 64 80 96 112 128
+					144 160 176 192 208 224 240 255>;
+		default-brightness-level = <9>;
```

Comment:
So without this property you'll have 255 brightness levels and now you're declaring 17 brightness levels so it's restricted to these led brightness values, right? Do these values (0, 16, 32, etc) come from downstream or did you just assign them yourself?

Reply:
You are right, and it looks like it doesn't need the brightness-levels property for default-brightness-levels to work. However there is another problem with display apparently not turning off according to dmesg. I'll mark this PR as WIP until I see if I can solve that.